vars/kola.groovy

```diff
@@ -21,9 +23,14 @@ def call(params = [:]) {
     def buildID = params.get('build', "latest");
     def arch = params.get('arch', "x86_64");
     def marker = params.get('marker', "");
-    def rerun = "--rerun"
-    if (params.get('disableRerun', false)) {
-        rerun = ""
+    def rerun = ""
+    if (!params.get('disableRerun', false)) {
+        rerun = "--rerun"
+        if (!params.get('allowRerunSuccess', false)) {
```

Comment:
Shouldn't this be

```suggestion
        if (params.get('allowRerunSuccess', true)) {
```

so we default to on?

Reply:
Alternatively, I think this is correct as is if we call it `disallowRerunSuccess`.